packages/cli/src/webpack/loaders/locale-loader.ts

```diff
@@ -0,0 +1,59 @@
+import fs from 'fs';
+import path from 'path';
+
+import MessageFormat from '@messageformat/core';
+import compileModule, {
+  StringStructure,
+} from '@messageformat/core/lib/compile-module';
+import globby from 'globby';
+import { LoaderDefinitionFunction } from 'webpack';
+
+import { ImaConfig } from '../../types';
+
+export interface LocaleLoaderOptions {
+  imaConfig?: ImaConfig;
+  rootDir?: string;
+}
+
+/**
+ * This loader load locale files of app and merged them to one compiled by messageFormat
+ *
+ * @param {string} locale Module source.
+ * @returns {string}
+ */
+const LocaleLoader: LoaderDefinitionFunction<LocaleLoaderOptions> = function (
+  locale
+) {
+  const { imaConfig, rootDir } = this.getOptions();
```

Comment:
add `this.cacheable(true);` so it can be cacheable using the filesystem cache.

Reply:
I thought that is default behaviour -  https://webpack.js.org/api/loaders/#thiscacheable

Reply:
my bad :)